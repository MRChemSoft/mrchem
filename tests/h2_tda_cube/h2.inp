{
    "world_prec": 0.00001,
    "world_size": 8,
    "MPI": {
        "numerically_exact": true
    },
    "Molecule": {
        "coords": "H      0.0000 0.0000   0.0000\nH      0.0000 0.0000    1.4\n"
    },
    "WaveFunction": {
        "method": "HF",
        "restricted": true
    },
    "Basis" : {
        "type" : "Interpolating",                  
        "order" : 7                             
    },
    "SCF": {
        "run": false,
        "guess_type": "mw",
        "kain": 5,
        "guess_prec": 0.0001

    },
    "Properties":{
        "excited_states": true
    },
    "ExcitedStates": {
        "n_states": 2,
        "run_tda": true,
        "kain":5,
        "max_iter":100
    },
    "Response": {
        "orbital_thrs": 0.0001,
        "property_thrs": 0.0001, 
        "guess_type": "cube",
        "guess_prec": 0.001
    }
}
