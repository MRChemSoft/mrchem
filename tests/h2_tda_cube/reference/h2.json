{
  "input": {
    "constants": {
      "angstrom2bohrs": 1.8897261246257702,
      "dipmom_au2debye": 2.5417464739297717,
      "electron_g_factor": -2.00231930436256,
      "fine_structure_constant": 0.0072973525693,
      "hartree2ev": 27.211386245988,
      "hartree2kcalmol": 627.5094740630558,
      "hartree2kjmol": 2625.4996394798254,
      "hartree2simagnetizability": 78.9451185,
      "hartree2wavenumbers": 219474.6313632,
      "light_speed": 137.035999084
    },
    "molecule": {
      "charge": 0,
      "coords": [
        {
          "atom": "h",
          "xyz": [
            0.0,
            0.0,
            0.0
          ]
        },
        {
          "atom": "h",
          "xyz": [
            0.0,
            0.0,
            1.4
          ]
        }
      ],
      "multiplicity": 1
    },
    "mpi": {
      "bank_size": -1,
      "numerically_exact": true,
      "shared_memory_size": 10000
    },
    "mra": {
      "basis_order": 7,
      "basis_type": "interpolating",
      "boxes": [
        2,
        2,
        2
      ],
      "corner": [
        -1,
        -1,
        -1
      ],
      "max_scale": 20,
      "min_scale": -7
    },
    "printer": {
      "file_name": "h2",
      "print_constants": false,
      "print_level": 0,
      "print_mpi": false,
      "print_prec": 6,
      "print_width": 75
    },
    "rsp_calculations": {
      "excited_states": {
        "fock_operator": {
          "coulomb_operator": {
            "poisson_prec": 1e-05,
            "shared_memory": false
          },
          "exchange_operator": {
            "exchange_prec": -1.0,
            "poisson_prec": 1e-05
          }
        },
        "run_tda": true,
        "states": [
          {
            "exc_solver": {
              "checkpoint": false,
              "file_chk_x": "checkpoint/X_rsp_0",
              "file_chk_y": "checkpoint/Y_rsp_0",
              "final_prec": 1e-05,
              "helmholtz_prec": -1.0,
              "kain": 5,
              "max_iter": 100,
              "method": "Hartree-Fock",
              "orbital_thrs": 0.0001,
              "orth_prec": 1e-14,
              "property_thrs": 0.0001,
              "start_prec": 1e-05
            },
            "initial_guess": {
              "file_CUBE_x_a": "cube_vectors/CUBE_x_a_exc_0_vector.json",
              "file_CUBE_x_b": "cube_vectors/CUBE_x_b_exc_0_vector.json",
              "file_CUBE_x_p": "cube_vectors/CUBE_x_p_exc_0_vector.json",
              "file_CUBE_y_a": "cube_vectors/CUBE_y_a_exc_0_vector.json",
              "file_CUBE_y_b": "cube_vectors/CUBE_y_b_exc_0_vector.json",
              "file_CUBE_y_p": "cube_vectors/CUBE_y_p_exc_0_vector.json",
              "file_chk_x": "checkpoint/X_exc_0",
              "file_chk_y": "checkpoint/Y_exc_0",
              "file_x_a": "initial_guess/X_a_exc_0",
              "file_x_b": "initial_guess/X_b_exc_0",
              "file_x_p": "initial_guess/X_p_exc_0",
              "file_y_a": "initial_guess/Y_a_exc_0",
              "file_y_b": "initial_guess/Y_b_exc_0",
              "file_y_p": "initial_guess/Y_p_exc_0",
              "prec": 0.001,
              "type": "cube"
            }
          },
          {
            "exc_solver": {
              "checkpoint": false,
              "file_chk_x": "checkpoint/X_rsp_1",
              "file_chk_y": "checkpoint/Y_rsp_1",
              "final_prec": 1e-05,
              "helmholtz_prec": -1.0,
              "kain": 5,
              "max_iter": 100,
              "method": "Hartree-Fock",
              "orbital_thrs": 0.0001,
              "orth_prec": 1e-14,
              "property_thrs": 0.0001,
              "start_prec": 1e-05
            },
            "initial_guess": {
              "file_CUBE_x_a": "cube_vectors/CUBE_x_a_exc_1_vector.json",
              "file_CUBE_x_b": "cube_vectors/CUBE_x_b_exc_1_vector.json",
              "file_CUBE_x_p": "cube_vectors/CUBE_x_p_exc_1_vector.json",
              "file_CUBE_y_a": "cube_vectors/CUBE_y_a_exc_1_vector.json",
              "file_CUBE_y_b": "cube_vectors/CUBE_y_b_exc_1_vector.json",
              "file_CUBE_y_p": "cube_vectors/CUBE_y_p_exc_1_vector.json",
              "file_chk_x": "checkpoint/X_exc_1",
              "file_chk_y": "checkpoint/Y_exc_1",
              "file_x_a": "initial_guess/X_a_exc_1",
              "file_x_b": "initial_guess/X_b_exc_1",
              "file_x_p": "initial_guess/X_p_exc_1",
              "file_y_a": "initial_guess/Y_a_exc_1",
              "file_y_b": "initial_guess/Y_b_exc_1",
              "file_y_p": "initial_guess/Y_p_exc_1",
              "prec": 0.001,
              "type": "cube"
            }
          }
        ],
        "unperturbed": {
          "fock_operator": {
            "coulomb_operator": {
              "poisson_prec": 1e-05,
              "shared_memory": false
            },
            "exchange_operator": {
              "exchange_prec": -1.0,
              "poisson_prec": 1e-05
            },
            "kinetic_operator": {
              "derivative": "abgv_55"
            },
            "nuclear_operator": {
              "proj_prec": 1e-05,
              "shared_memory": false,
              "smooth_prec": 1e-05
            }
          },
          "localize": false,
          "precision": 1e-05
        }
      }
    },
    "scf_calculation": {
      "fock_operator": {
        "coulomb_operator": {
          "poisson_prec": 1e-05,
          "shared_memory": false
        },
        "exchange_operator": {
          "exchange_prec": -1.0,
          "poisson_prec": 1e-05
        },
        "kinetic_operator": {
          "derivative": "abgv_55"
        },
        "nuclear_operator": {
          "proj_prec": 1e-05,
          "shared_memory": false,
          "smooth_prec": 1e-05
        }
      },
      "initial_guess": {
        "environment": "None",
        "external_field": "None",
        "file_CUBE_a": "cube_vectors/CUBE_a_vector.json",
        "file_CUBE_b": "cube_vectors/CUBE_b_vector.json",
        "file_CUBE_p": "cube_vectors/CUBE_p_vector.json",
        "file_basis": "initial_guess/mrchem.bas",
        "file_chk": "checkpoint/phi_scf",
        "file_gto_a": "initial_guess/mrchem.moa",
        "file_gto_b": "initial_guess/mrchem.mob",
        "file_gto_p": "initial_guess/mrchem.mop",
        "file_phi_a": "initial_guess/phi_a_scf",
        "file_phi_b": "initial_guess/phi_b_scf",
        "file_phi_p": "initial_guess/phi_p_scf",
        "localize": false,
        "method": "Hartree-Fock",
        "prec": 0.0001,
        "relativity": "None",
        "restricted": true,
        "screen": 12.0,
        "type": "mw",
        "zeta": 0
      },
      "properties": {
        "dipole_moment": {
          "dip-1": {
            "operator": "h_e_dip",
            "precision": 1e-05,
            "r_O": [
              0.0,
              0.0,
              0.0
            ]
          }
        }
      }
    },
    "schema_name": "mrchem_input",
    "schema_version": 1
  },
  "output": {
    "properties": {
      "center_of_mass": [
        0.0,
        0.0,
        0.7
      ],
      "charge": 0,
      "dipole_moment": {
        "dip-1": {
          "magnitude": 4.122385589338093e-06,
          "r_O": [
            0.0,
            0.0,
            0.0
          ],
          "vector": [
            4.3905426009868324e-12,
            4.280409125783221e-12,
            -4.122385589333533e-06
          ],
          "vector_el": [
            0.0,
            0.0,
            -1.4000041223888764
          ],
          "vector_nuc": [
            4.396384732509632e-12,
            4.286774492689872e-12,
            1.400000000003287
          ]
        }
      },
      "excitation_energies": {
        "state-1": 0.46814789980337057,
        "state-2": 0.47798185348703426
      },
      "geometry": [
        {
          "symbol": "H",
          "xyz": [
            0.0,
            0.0,
            0.0
          ]
        },
        {
          "symbol": "H",
          "xyz": [
            0.0,
            0.0,
            1.4
          ]
        }
      ],
      "multiplicity": 1,
      "orbital_energies": {
        "energy": [
          -0.5947842934625274
        ],
        "occupation": [
          2.0
        ],
        "spin": [
          "p"
        ],
        "sum_occupied": -1.1895685869250547
      },
      "scf_energy": {
        "E_ee": 1.3169740976622437,
        "E_eext": 0.0,
        "E_el": -1.8480281528851497,
        "E_en": -3.6325425523111403,
        "E_kin": 1.1260549247762603,
        "E_next": 0.0,
        "E_nn": 0.7142857142857143,
        "E_nuc": 0.7142857142857143,
        "E_tot": -1.1337424385994352,
        "E_x": -0.6585146230125135,
        "E_xc": 0.0,
        "Er_el": 0.0,
        "Er_nuc": 0.0,
        "Er_tot": 0.0
      }
    },
    "provenance": {
      "creator": "MRChem",
      "mpi_processes": 1,
      "nthreads": 36,
      "routine": "mrchem.x",
      "total_cores": 36,
      "version": "1.2.0-alpha"
    },
    "rsp_calculations": {
      "excited_states": {
        "states": [
          {
            "exc_solver": {
              "converged": true,
              "cycles": [
                {
                  "frequency": 0.46818063363743406,
                  "frequency_update": -0.0027488435910707487,
                  "mo_residual": 0.01891605780124631,
                  "wall_time": 2.483328623
                },
                {
                  "frequency": 0.46815897463769857,
                  "frequency_update": -2.1658999735496565e-05,
                  "mo_residual": 0.004398391427829538,
                  "wall_time": 3.135665478
                },
                {
                  "frequency": 0.46815043733891754,
                  "frequency_update": -8.5372987810306e-06,
                  "mo_residual": 0.0022687537176325407,
                  "wall_time": 3.413425175
                },
                {
                  "frequency": 0.46814808747683473,
                  "frequency_update": -2.3498620828066308e-06,
                  "mo_residual": 0.0012591698265178217,
                  "wall_time": 3.49208684
                },
                {
                  "frequency": 0.4681479046836517,
                  "frequency_update": -1.8279318303449088e-07,
                  "mo_residual": 0.0002924887299506004,
                  "wall_time": 3.684507502
                },
                {
                  "frequency": 0.46814789980337057,
                  "frequency_update": -4.880281123664787e-09,
                  "mo_residual": 9.2479455427704e-05,
                  "wall_time": 3.67550207
                }
              ],
              "frequency": 0.46814789980337057,
              "wall_time": 21.794138141
            }
          },
          {
            "exc_solver": {
              "converged": true,
              "cycles": [
                {
                  "frequency": 0.478019376059328,
                  "frequency_update": -0.002578957366675405,
                  "mo_residual": 0.03730052607643635,
                  "wall_time": 2.754913854
                },
                {
                  "frequency": 0.4779856801198373,
                  "frequency_update": -3.3695939490707705e-05,
                  "mo_residual": 0.010218073097048468,
                  "wall_time": 3.313607226
                },
                {
                  "frequency": 0.4779824540646557,
                  "frequency_update": -3.226055181548837e-06,
                  "mo_residual": 0.003013382011179116,
                  "wall_time": 3.587167591
                },
                {
                  "frequency": 0.47798191200486223,
                  "frequency_update": -5.420597934957527e-07,
                  "mo_residual": 0.001026263280301664,
                  "wall_time": 3.7179219
                },
                {
                  "frequency": 0.47798186008736415,
                  "frequency_update": -5.1917498078424273e-08,
                  "mo_residual": 0.00027722811339788605,
                  "wall_time": 3.937174099
                },
                {
                  "frequency": 0.47798185348703426,
                  "frequency_update": -6.600329893746704e-09,
                  "mo_residual": 8.777551041455796e-05,
                  "wall_time": 3.979247447
                }
              ],
              "frequency": 0.47798185348703426,
              "wall_time": 23.331933032
            }
          }
        ],
        "success": true
      }
    },
    "scf_calculation": {
      "initial_energy": {
        "E_ee": 1.3169740976622437,
        "E_eext": 0.0,
        "E_el": -1.8480281528851497,
        "E_en": -3.6325425523111403,
        "E_kin": 1.1260549247762603,
        "E_next": 0.0,
        "E_nn": 0.7142857142857143,
        "E_nuc": 0.7142857142857143,
        "E_tot": -1.1337424385994352,
        "E_x": -0.6585146230125135,
        "E_xc": 0.0,
        "Er_el": 0.0,
        "Er_nuc": 0.0,
        "Er_tot": 0.0
      },
      "success": true
    },
    "schema_name": "mrchem_output",
    "schema_version": 1,
    "success": true
  }
}
