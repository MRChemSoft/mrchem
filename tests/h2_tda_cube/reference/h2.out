                                                                           
                                                                           
***************************************************************************
***                                                                     ***
***                                                                     ***
***          __  __ ____   ____ _                                       ***
***         |  \/  |  _ \ / ___| |__   ___ _ __ ___                     ***
***         | |\/| | |_) | |   | '_ \ / _ \ '_ ` _ \                    ***
***         | |  | |  _ <| |___| | | |  __/ | | | | |                   ***
***         |_|  |_|_| \_\\____|_| |_|\___|_| |_| |_|                   ***
***                                                                     ***
***         VERSION            1.2.0-alpha                              ***
***                                                                     ***
***         Git branch         excited_states_solver                    ***
***         Git commit hash    837b038e35545bdeead6                     ***
***         Git commit author  Gabriel Gerez                            ***
***         Git commit date    Wed Jan 18 17:37:06 2023 +0100           ***
***                                                                     ***
***         Contact: luca.frediani@uit.no                               ***
***                                                                     ***
***         Radovan Bast            Magnar Bjorgve                      ***
***         Roberto Di Remigio      Antoine Durdek                      ***
***         Luca Frediani           Gabriel Gerez                       ***
***         Stig Rune Jensen        Jonas Juselius                      ***
***         Rune Monstad            Peter Wind                          ***
***                                                                     ***
***************************************************************************

---------------------------------------------------------------------------

 MPI processes           :       (no bank)                               1
 OpenMP threads          :                                              36
 Total cores             :                                              36
                                                                           
---------------------------------------------------------------------------

XCFun DFT library Copyright 2009-2020 Ulf Ekstrom and contributors.
See http://dftlibs.org/xcfun/ for more information.

This is free software; see the source code for copying conditions.
There is ABSOLUTELY NO WARRANTY; not even for MERCHANTABILITY or
FITNESS FOR A PARTICULAR PURPOSE. For details see the documentation.
Scientific users of this library should cite
U. Ekstrom, L. Visscher, R. Bast, A. J. Thorvaldsen and K. Ruud;
J.Chem.Theor.Comp. 2010, DOI: 10.1021/ct100117s

---------------------------------------------------------------------------

 MRCPP version         : 1.4.1
 Git branch            : HEAD
 Git commit hash       : 75d41879b1908a94a452
 Git commit author     : Stig Rune Jensen
 Git commit date       : Thu Jan 6 11:38:53 2022 +0100

 Linear algebra        : EIGEN v3.4.0
 Parallelization       : OpenMP (36 threads)

---------------------------------------------------------------------------


                                                                           
===========================================================================
                         MultiResolution Analysis
---------------------------------------------------------------------------
 polynomial order      : 7
 polynomial type       : Interpolating
---------------------------------------------------------------------------
 total boxes           : 8
 boxes                 : [          2           2           2 ]
 unit lengths          : [  128.00000   128.00000   128.00000 ]
 scaling factor        : [    1.00000     1.00000     1.00000 ]
 lower bounds          : [ -128.00000  -128.00000  -128.00000 ]
 upper bounds          : [  128.00000   128.00000   128.00000 ]
 total length          : [  256.00000   256.00000   256.00000 ]
===========================================================================


                                                                           
***************************************************************************
***                                                                     ***
***                        Initializing Molecule                        ***
***                                                                     ***
***************************************************************************
                                                                           
                                                                           
===========================================================================
                                 Molecule
---------------------------------------------------------------------------
 Charge                  :                                               0
 Multiplicity            :                                               1
---------------------------------------------------------------------------
    N    Atom            :               x               y               z
---------------------------------------------------------------------------
    0       H            :        0.000000        0.000000        0.000000
    1       H            :        0.000000        0.000000        1.400000
---------------------------------------------------------------------------
 Center of mass          :        0.000000        0.000000        0.700000
===========================================================================


                                                                           
***************************************************************************
***                                                                     ***
***                Computing Initial Guess Wavefunction                 ***
***                                                                     ***
***************************************************************************
                                                                           
                                                                           
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 Calculation             : Compute initial orbitals
 Method                  : Project MW molecular orbitals
 Precision               : 1.00000e-04
 Restricted              : True
 MO file                 : initial_guess/phi_p_scf
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~


===========================================================================
                            Molecular Orbitals
---------------------------------------------------------------------------
 Alpha electrons         :                                               1
 Beta electrons          :                                               1
 Total electrons         :                                               2
---------------------------------------------------------------------------
    n  Occ Spin          :                                            Norm
---------------------------------------------------------------------------
    0    2    p          :                              1.000000038380e+00
===========================================================================


~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 Calculation             : Compute initial energy
 Method                  : Hartree-Fock
 Relativity              : None
 Environment             : None
 External fields         : None
 Precision               : 1.00000e-04
 Localization            : Off
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~


===========================================================================
                        Molecular Energy (initial)
---------------------------------------------------------------------------
 Kinetic energy          :            (au)                  1.126054924776
 E-N energy              :            (au)                 -3.632542552311
 Coulomb energy          :            (au)                  1.316974097662
 Exchange energy         :            (au)                 -0.658514623013
 X-C energy              :            (au)                  0.000000000000
 N-N energy              :            (au)                  0.714285714286
---------------------------------------------------------------------------
 Electronic energy       :            (au)                 -1.848028152885
 Nuclear energy          :            (au)                  0.714285714286
---------------------------------------------------------------------------
 Total energy            :            (au)             -1.133742438599e+00
                         :      (kcal/mol)             -7.114341213685e+02
                         :        (kJ/mol)             -2.976640363806e+03
                         :            (eV)             -3.085070340020e+01
===========================================================================


===========================================================================
                        Orbital Energies (initial)
---------------------------------------------------------------------------
    n  Occ Spin          :                                         Epsilon
---------------------------------------------------------------------------
    0    2    p          :            (au)                 -0.594784293463
---------------------------------------------------------------------------
 Sum occupied            :            (au)                 -1.189568586925
===========================================================================


                                                                           
***************************************************************************
***                                                                     ***
***           Optimizing Transition Orbitals and Frequencies            ***
***                                                                     ***
***************************************************************************
                                                                           
                                                                           
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 Calculation             : Compute initial orbitals
 Method                  : Project cube file molecular orbitals
 Precision               : 1.00000e-03
 Restricted              : True
 MO file                 : cube_vectors/CUBE_x_p_exc_0_vector.json
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~


===========================================================================
                            Molecular Orbitals
---------------------------------------------------------------------------
 Alpha electrons         :                                               1
 Beta electrons          :                                               1
 Total electrons         :                                               2
---------------------------------------------------------------------------
    n  Occ Spin          :                                            Norm
---------------------------------------------------------------------------
    0    2    p          :                              9.957811825833e-01
===========================================================================


~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 Calculation             : Optimize Transition Orbitals and Frequencies
 Guess frequency         : 0.47093 au
 Method                  : Hartree-Fock
 Relativity              : None
 Checkpointing           : Off
 Max iterations          : 100
 KAIN solver             : 5
 Start precision         : 1.00000e-05
 Final precision         : 1.00000e-05
 Helmholtz precision     : Dynamic
 Property threshold      : 1.00000e-04
 Orbital threshold       : 1.00000e-04
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~


===========================================================================
 Iter           MO residual     Excited state energy                Update
---------------------------------------------------------------------------
    0          1.000000e+00           0.470929477229          4.709295e-01
    1          1.891606e-02           0.468180633637         -2.748844e-03
    2          4.398391e-03           0.468158974638         -2.165900e-05
    3          2.268754e-03           0.468150437339         -8.537299e-06
    4          1.259170e-03           0.468148087477         -2.349862e-06
    5          2.924887e-04           0.468147904684         -1.827932e-07
    6          9.247946e-05           0.468147899803         -4.880281e-09
---------------------------------------------------------------------------
                      SCF converged in 6 iterations!
===========================================================================


===========================================================================
                        Exciation energy (state 1)
---------------------------------------------------------------------------
 Wavelength              :            (nm)                       97.326833
 Frequency               :            (au)                        0.468148
                         :          (cm-1)                   102746.587733
---------------------------------------------------------------------------
===========================================================================


~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 Calculation             : Compute initial orbitals
 Method                  : Project cube file molecular orbitals
 Precision               : 1.00000e-03
 Restricted              : True
 MO file                 : cube_vectors/CUBE_x_p_exc_1_vector.json
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~


===========================================================================
                            Molecular Orbitals
---------------------------------------------------------------------------
 Alpha electrons         :                                               1
 Beta electrons          :                                               1
 Total electrons         :                                               2
---------------------------------------------------------------------------
    n  Occ Spin          :                                            Norm
---------------------------------------------------------------------------
    0    2    p          :                              9.974310292951e-01
===========================================================================


~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 Calculation             : Optimize Transition Orbitals and Frequencies
 Guess frequency         : 0.48060 au
 Method                  : Hartree-Fock
 Relativity              : None
 Checkpointing           : Off
 Max iterations          : 100
 KAIN solver             : 5
 Start precision         : 1.00000e-05
 Final precision         : 1.00000e-05
 Helmholtz precision     : Dynamic
 Property threshold      : 1.00000e-04
 Orbital threshold       : 1.00000e-04
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~


===========================================================================
 Iter           MO residual     Excited state energy                Update
---------------------------------------------------------------------------
    0          1.000000e+00           0.480598333426          4.805983e-01
    1          3.730053e-02           0.478019376059         -2.578957e-03
    2          1.021807e-02           0.477985680120         -3.369594e-05
    3          3.013382e-03           0.477982454065         -3.226055e-06
    4          1.026263e-03           0.477981912005         -5.420598e-07
    5          2.772281e-04           0.477981860087         -5.191750e-08
    6          8.777551e-05           0.477981853487         -6.600330e-09
---------------------------------------------------------------------------
                      SCF converged in 6 iterations!
===========================================================================


===========================================================================
                        Exciation energy (state 2)
---------------------------------------------------------------------------
 Wavelength              :            (nm)                       95.324440
 Frequency               :            (au)                        0.477982
                         :          (cm-1)                   104904.891092
---------------------------------------------------------------------------
===========================================================================


                                                                           
***************************************************************************
***                                                                     ***
***                    Printing Molecular Properties                    ***
***                                                                     ***
***************************************************************************
                                                                           
                                                                           
===========================================================================
                                 Molecule
---------------------------------------------------------------------------
 Charge                  :                                               0
 Multiplicity            :                                               1
---------------------------------------------------------------------------
    N    Atom            :               x               y               z
---------------------------------------------------------------------------
    0       H            :        0.000000        0.000000        0.000000
    1       H            :        0.000000        0.000000        1.400000
---------------------------------------------------------------------------
 Center of mass          :        0.000000        0.000000        0.700000
===========================================================================


===========================================================================
                         Molecular Energy (final)
---------------------------------------------------------------------------
 Kinetic energy          :            (au)                  1.126054924776
 E-N energy              :            (au)                 -3.632542552311
 Coulomb energy          :            (au)                  1.316974097662
 Exchange energy         :            (au)                 -0.658514623013
 X-C energy              :            (au)                  0.000000000000
 N-N energy              :            (au)                  0.714285714286
---------------------------------------------------------------------------
 Electronic energy       :            (au)                 -1.848028152885
 Nuclear energy          :            (au)                  0.714285714286
---------------------------------------------------------------------------
 Total energy            :            (au)             -1.133742438599e+00
                         :      (kcal/mol)             -7.114341213685e+02
                         :        (kJ/mol)             -2.976640363806e+03
                         :            (eV)             -3.085070340020e+01
===========================================================================


===========================================================================
                         Orbital Energies (final)
---------------------------------------------------------------------------
    n  Occ Spin          :                                         Epsilon
---------------------------------------------------------------------------
    0    2    p          :            (au)                 -0.594784293463
---------------------------------------------------------------------------
 Sum occupied            :            (au)                 -1.189568586925
===========================================================================


===========================================================================
                        Excitation Energies (final)
---------------------------------------------------------------------------
    r                    :                                           Omega
---------------------------------------------------------------------------
    1                    :            (au)                  0.468147899803
    2                    :            (au)                  0.477981853487
===========================================================================


===========================================================================
                           Dipole Moment (dip-1)
---------------------------------------------------------------------------
 r_O                     :        0.000000        0.000000        0.000000
---------------------------------------------------------------------------
 Electronic vector       :       -0.000000       -0.000000       -1.400004
 Magnitude               :            (au)                        1.400004
                         :         (Debye)                        3.558456
---------------------------------------------------------------------------
 Nuclear vector          :        0.000000        0.000000        1.400000
 Magnitude               :            (au)                        1.400000
                         :         (Debye)                        3.558445
---------------------------------------------------------------------------
 Total vector            :        0.000000        0.000000       -0.000004
 Magnitude               :            (au)                        0.000004
                         :         (Debye)                        0.000010
===========================================================================


                                                                           
                                                                           
***************************************************************************
***                                                                     ***
***                            Exiting MRChem                           ***
***                                                                     ***
***                       Wall time :  0h  0m 48s                       ***
***                                                                     ***
***************************************************************************
                                                                           
                                                                           
